README.md

```diff
@@ -18,13 +18,32 @@ The operator image needs to be pushed to a remote repository:
 ```shell script
 podman push quay.io/<insert username>/wmco:$VERSION_TAG
 ```
-
-## Testing locally
+## Development workflow
+To run the operator on a cluster (for testing purpose only).
 To run the e2e tests for WMCO locally against an OpenShift cluster set up on AWS, we need to setup the following environment variables.
 ```shell script
 export KUBECONFIG=<path to kubeconfig>
 export AWS_SHARED_CREDENTIALS_FILE=<path to aws credentials file>
 export KUBE_SSH_KEY_PATH=<path to RSA type ssh key>
+```
+
+To run the operator on a cluster, use: 
+```shell script
+hack/olm.sh run -c "<OPERATOR_IMAGE>"
+```
```

Comment:
Call out that the above step will build and push the image, and the `Build` steps are not required.